app/aem/core/src/main/java/com/cognifide/apm/core/endpoints/ScriptExecutionServlet.java

```diff
@@ -77,12 +77,16 @@ protected void doGet(SlingHttpServletRequest request, SlingHttpServletResponse r
         return ResponseEntity.ok("Script successfully executed")
             .addEntry("status", status.getStatus())
             .addEntry("output", ((ExecutionStatus.FinishedSuccessfulExecution) status).getEntries())
-            .addEntry("path", ((ExecutionStatus.FinishedSuccessfulExecution) status).getPath());
+            .addEntry("path", ((ExecutionStatus.FinishedSuccessfulExecution) status).getPath())

```

Comment:
why these casts are neeed here?

Reply:
when the type is one and you are casting... why this direct type cannot be used here?